hw/ip/entropy_src/data/entropy_src_testplan.hjson

```diff
@@ -39,40 +52,73 @@
       name: rng_mode
       desc: '''
             Verify rng entropy
-            - Various clk speeds
-            - Various FIFO depths
+            - Random FIFO depths
+	    - Random rates
```

Comment:
nit: misaligned

Reply:
Never mind this is only in the git presentation not in the actual file.. :S

Reply:
It will be great to mass search and replace tabs with spaces. It is annoying to see it show up as diffs. It is indeed a part of our style guide:
https://github.com/lowRISC/style-guides/blob/master/VerilogCodingStyle.md#no-tabs

This should actually apply to ALL files.